src/common/pick_address.cc

```diff
@@ -207,7 +207,7 @@ std::string pick_iface(CephContext *cct, const struct sockaddr_storage &network)
     return {};
   }
 
-  unsigned int prefix_len = 0;
+  unsigned int prefix_len = 128U;
```

Comment:
i'd suggest replace the magic number with
`max(sizeof(in_addr::s_addr), sizeof(in6_addr::s6_addr)) * CHAR_BIT`. and also make it a const.

Reply:
Thank you for your suggestion. Done.